promscale/visualize-data/jaeger.md

```diff
@@ -0,0 +1,67 @@
+# Visualise Promscale traces in Jaeger
+This section shows you how to integrate
+[Jaeger][jaeger-ui] with Promscale.
+
+To visualize traces in Jaeger with Promscale, you need to have Jaeger query component running that queries and visualizes traces from Promscale. 
+
+<procedure>
+
+## Connect Jaeger query with Promscale
+
+1.  Deploy the Jaeger Query component from the Jaeger 
+    [deployments page][jaeger-deployments], if you aren't already running the Jaeger query.
+    
+    **Note**: For Jaeger to visualize traces in Promscale. We need Jaeger Query of version `1.30` or above. 
+    
+1.  Set the Jaeger query configuration through environment variables 
+    as `SPAN_STORAGE_TYPE`  to `grpc-plugin` and 
+    `GRPC_STORAGE_SERVER` to Promscale gRPC endpoint `<PROMSCALE_HOST>:9202` before 
+    Jaeger query startup, with this the Jaeger query configures the 
+    span storage type as gRPC server and gRPC storage server endpoint as Promscale.
+    ```
+    SPAN_STORAGE_TYPE=grpc-plugin
+    GRPC_STORAGE_SERVER=<PROMSCALE_HOST>:9202
+    ```
+    Here `9202` is the default gRPC server port in Promscale.
+1.  Now on Jaeger query startup, it should be successfully connected with 
+    Promscale to visualize the traces. 
+
+</procedure>
+
+Visualising traces from Promscale in Jaeger query using the filters offered on the left search menu
```

Comment:
```suggestion
Visualizing traces from Promscale in Jaeger query using the filters offered on the left search menu
```

This sentence is incomplete. What are you trying to say here?

Reply:
This is a short description for an image after this phrase.